Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/DemoTest.cs

```diff
@@ -27,7 +25,7 @@ public void Setup()
         }
 
         [TearDown]
-        public void AfterEach()
+        public new void AfterEach()
         {
             AqualityServices.Browser.Quit();
```

Comment:
we should quit the browser only if it was started. And we should do this in the BaseTest probably.

        [TearDown]
        public void CleanUp()
        {
            if (AqualityServices.IsBrowserStarted)
            {
                AqualityServices.Browser.Quit();
            }
        }

Reply:
It is necessary to discuss.